java/src/jmri/jmrit/logix/MergePrompt.java

```diff
@@ -127,7 +128,6 @@ public void windowClosing(java.awt.event.WindowEvent e) {
         Dimension screen = getToolkit().getScreenSize();
         setLocation(screen.width / 3, screen.height / 4);
         setAlwaysOnTop(true);
-        setVisible(true);
```

Comment:
Why is this line removed? Is it guaranteed that `setVisible()` is always called anyway?

Reply:
I noticed that the constructor is package private so it's no risk of it being called outside of this package.